exercises/01-getting-started.md

```diff
@@ -1,63 +1,81 @@
 # Getting started
-In the following we describe how to get started with this project.
+
 You will create a new **private** repository based on this repository and run the application locally to get familiar with it.
 
 > In the original session, the participants were handed out a participantId. If you go through the tasks on your own, make up your own id. We recommend to refrain from using personal data.
 
 ## Create a new repository based on this template
-Click the *Use this template* button on the top of the GitHub page.
+
+Click `Use this template` on the top of the GitHub page.
 ![Use Template](images/use-template.png)
 
-Fill in the name of your new repository (e. g. *cloud-sdk-team-calendar*) and make sure to create a **private** repository. Click on *Create repository*.
+Fill in the name of your new repository (e. g. _cloud-sdk-team-calendar_) and make sure to create a **private** repository. Click on `Create repository`.
 ![Create New](images/create-new.png)
 
 > ### Why private?
-> We recommend to create your project as a private repository due to licensing constraints. In [exercise 5](05-generate-odata-client.md) we will generate an OData client for SAP SuccessFactors, that is subject to SAP intellectual property and should not be made available as open source. As long as you don't publish these changes, feel free to create a public repository as well.
+>
+> We recommend to create your project as a private repository due to licensing constraints. In [exercise 2](02-generate-odata-clients.md) we will generate OData clients for SAP S/4HANA and SAP SuccessFactors, that is subject to SAP intellectual property and should not be made available as open source. As long as you don't publish these changes, feel free to create a public repository as well.
```

Comment:
Q: Is this still correct? We added the pre-generated clients to the repo meanwhile (with the legal notice on copyright), so I think this is not required anymore. cc @marikaner

Reply:
As I discussed with @marikaner and @deekshas8, I removed the notion of the private repo.